ext/flexmailer/info.xml

```diff
@@ -14,16 +14,16 @@
     <url desc="Support">http://civicrm.stackexchange.com/</url>
     <url desc="Licensing">http://www.gnu.org/licenses/agpl-3.0.html</url>
   </urls>
-  <releaseDate>2019-11-26</releaseDate>
-  <version>1.1.1</version>
-  <develStage>alpha</develStage>
+  <releaseDate>2020-08-05</releaseDate>
+  <version>1.1.2</version>
+  <develStage>stable</develStage>
   <comments>
     FlexMailer is an email delivery engine which replaces the internal guts
     of CiviMail.  It is a drop-in replacement which enables *other* extensions
     to provide richer email features.
   </comments>
   <compatibility>
-    <ver>5.13</ver>
+    <ver>5.29</ver>
```

Comment:
is there a reason for this?

Reply:
Well that is the version it will ship with core with so it seems to make sense to align it  - we expect it to be matched to core going forwards & that people won't use this version with other releases of core

Reply:
ie if we add a function to core that we call from this extension part of shipping with core is that we can do that.

Reply:
Note it probably works fine with versions before 5.29 - but 5.29 is the version we warrant & maintain this version against and we have no intention of testing that it works on any earlier version